core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -459,14 +465,24 @@ class WorkflowResource {
   @Path("/search")
   def searchWorkflows(
```

Comment:
Please remove unused code instead of commenting them out.

Reply:
done